build.properties

```diff
@@ -204,6 +204,7 @@
     #    core,\
     #    util
 
+    build.lib.versions.skip.read.only=true
```

Comment:
You're skipping the read only repos so a merge pull from `com-liferay-apio-architect` with a new 3rd-party lib won't fail, right? But it's not nice to simply exclude read only repos 😞

I wonder if the merge pull could re-run `ant build-lib-versions` and commit...

Reply:
CI also copies over `private` modules from `master-private`, so the versions files need to be updated whenever the `git-commit` file is updated too.

I'll take a look and see if we can have a separate server automatically update the versions files once a day like the `Language.properties` translations.

Reply:
But we should probably only consider the public modules in `liferay-portal`, and both public and private in private branches of `liferay-portal-ee`.